packages/services/server/README.md

```diff
@@ -10,7 +10,6 @@ The GraphQL API for GraphQL Hive.
 | `ENCRYPTION_SECRET`                                  | **Yes**                                            | Secret for encrypting stuff.                                                                  | `8ebe95cg21c1fee355e9fa32c8c33141`                        |
 | `WEB_APP_URL`                                        | **Yes**                                            | The url of the web app.                                                                       | `http://127.0.0.1:3000`                                   |
 | `GRAPHQL_PUBLIC_ORIGIN`                              | **Yes**                                            | The origin of the GraphQL server.                                                             | `http://127.0.0.1:4013`                                   |
-| `RATE_LIMIT_ENDPOINT`                                | **Yes**                                            | The endpoint of the rate limiting service.                                                    | `http://127.0.0.1:4012`                                   |
```

Comment:
Should this be replaced by one for the `COMMERCE_ENDPOINT`?

Reply:
It's in the `Hive Cloud Configuration` below